sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Storage/BlobsCheckpointStore.cs

```diff
@@ -107,15 +116,17 @@ await foreach (BlobItem blob in ContainerClient.GetBlobsAsync(traits: BlobTraits
                     ));
                 }
 
+                Logger.ListOwnershipAsyncComplete(fullyQualifiedNamespace, eventHubName, consumerGroup, ownershipList.Count);
```

Comment:
We may want to consider wrapping in a `try/finally`  to ensure that the complete event always gets emitted.   I don't have a very strong opinion there, but that's the approach we used for the main Event Hubs library.   You'll always see the "Start" and "Complete" and may or may not see an "Error" between them.